src/views/documents/DocumentsPrintingView.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <div class="section documents-view">
+    <html-header :title="getDocumentTypeTitle(documentType)" />
+    <printing-options
+      :documents="documents"
+      :promises="promises"
+      @set-page-breaks="pageBreaks = $event"
+      @set-image-printing="includeImages = $event"
+      @set-summary-visibility="summaryVisibility = $event"
+    />
+    <printing-summary
+      v-if="summaryVisibility"
+      :documents="documents"
+      :document-type="documentType"
+      @summary-click="scrollTo($event)"
+    />
```

Comment:
I wonder if there could be a way to use columns for displaying pagination, options and print button and use less vertical space?
But it could be for another PR.